tests/sonic_xcvr/test_sff8472.py

```diff
@@ -22,7 +22,7 @@ class TestSff8472(object):
 
     def test_api(self):
         """
-        Verify all api access valid fields    
```

Comment:
Can you please revert this and other unintended changes in this file?

Reply:
Already reverted, please refresh